cjworkbench/tests/i18n/test_trans.py

```diff
@@ -72,23 +116,32 @@ def test_trans_params(self):
             "Hello you there {c}!",
         )
 
-    # Tests that a programmer will get an exception when including a numeric variable in the message
+    # Tests that a programmer can include a numeric variable in the message
     def test_format_invalid_default(self):
-        with self.assertRaises(InvalidICUParameters):
+        self.assertEqual(
             trans_html(
                 default_locale,
                 mock_message_id,
                 default="Hello {a} {0} {b}",
                 parameters={"a": "you", "0": "!", "b": "2"},
             ),
+            "Hello you ! 2",
+        )
 
     # Tests that a translator can't break our system by including a numeric variable in the message
     def test_format_invalid_message(self):
-        self.assertEqual(
+        self.assertTrue(
```

Comment:
Also: please test that we raise `icu.ICUError` when given an invalid message format.

```
>>> mf = icu.MessageFormat("Hello, {person person person}!", icu.Locale.createFromName("en"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
icu.ICUError: ('Syntax error in format pattern', 0, 8, 'Hello, {\x00\udc8c羘\x00藐\udc86羘\x00', 'person person p\x00'), error code: 65799
```